### ISSUE TITLE
OpenGL is not working on Windows

### ISSUE THREAD

**Issue by niemeyer** [go-qml/qml#62]
This is being fixed by the refactoring of the gl API that is in progress.

**Comment by neclepsio** [comment]
Any estimation of when the refactoring will be ready? Since the current version does not have OpenGL ES 2.0 entry points, and it seems I cannot mix with other Go/OpenGL interfaces, my work is substantially on standby. Thank you.

**Comment by niemeyer** [comment]
That's the main task I'm working on, but I've been traveling for the last couple of weeks. I'm hoping I can land it over the next two weeks.

If you want to get out of standby right away easily, you can install Ubuntu on a machine or VM. Then, once the work is finished, you can move back to Windows.

**Comment by neclepsio** [comment]
The problem is not Windows, I've managed to correctly link everything (I sent you an email on how). The problem is the version of OpenGL the current implementation targets: I need to use version 2.0. I will wait, working with go-gl and glfw, until I will be able to merge the OpenGL part wot the rest. Thank you.

**Comment by niemeyer** [comment]
Sounds good, thanks for your patience.

**Comment by niemeyer** [comment]
The new OpenGL API is still progress, but is now live and should work on Windows. If it doesn't, please open a separate ticket with details.
